app/utils/file/index.ts

```diff
@@ -531,7 +531,6 @@ export const getAllFilesInCachesDirectory = async (serverUrl: string) => {
             totalSize,
         };
     } catch (error) {
-        logError('Failed getAllFilesInCachesDirectory', error);
```

Comment:
why the removal of this log ?

Reply:
whenever it reads the cache folder...if that folder is not found... it errors out and it is caught there.  however, this creates a false sense of 'error' as this is what we are telling it to do.  So, to reduce "noise",  I removed it.